pkg/uroot/uroot_test.go

```diff
@@ -0,0 +1,125 @@
+package uroot
+
+import (
+	"fmt"
+	"github.com/u-root/u-root/pkg/golang"
+	"go/build"
+	"path/filepath"
+	"reflect"
+	"testing"
+)
+
+func TestResolvePackagePaths(t *testing.T) {
+	defaultEnv := golang.Environ{Context: build.Default}
+	gopath1, _ := filepath.Abs("test/gopath1")
```

Comment:
check the error

(this will not fail in open source, but it will fail internally, but that'll be my problem)

Reply:
Done.